src/NServiceBus.FileBasedRouting/XmlRoutingFileParser.cs

```diff
@@ -42,23 +45,32 @@ public IEnumerable<EndpointRoutingConfiguration> Parse(XDocument document)
             return configs;
         }
 
-        static IEnumerable<Type> GetAllMessageTypes(XElement endpointElement, string singular, string plural)
+        IEnumerable<Type> GetAllMessageTypes(XElement endpointElement, string singular, string plural)
         {
             var handles = endpointElement.Element("handles");
 
             var separatelyConfiguredCommands = handles
                 ?.Elements(singular)
                 .Select(e => SelectCommand(e.Attribute("type").Value))
+                .Where(e => e != null)
```

Comment:
@timbussmann I'm still wondering why a `warn` would not work in the case of an spelling issue? How would you validate the endpoint name? Is there way to ensure there's spelling mistake there? Why is the assembly/type different?